examples/deepchecks_data_validation/run.py

```diff
@@ -35,13 +34,3 @@
         model_drift_detector=model_drift_detector,
     )
     pipeline_instance.run()
-
-    last_run = pipeline_instance.get_runs()[0]
-    data_val_step = last_run.get_step(step="data_validator")
-    model_val_step = last_run.get_step(step="model_validator")
-    data_drift_step = last_run.get_step(step="data_drift_detector")
-    model_drift_step = last_run.get_step(step="model_drift_detector")
-    DeepchecksVisualizer().visualize(data_val_step)
-    DeepchecksVisualizer().visualize(model_val_step)
-    DeepchecksVisualizer().visualize(data_drift_step)
-    DeepchecksVisualizer().visualize(model_drift_step)
```

Comment:
I greatly appreciate it that you see it this way as well. I was considering to support the old behavior but took it out since it felt redundant as you pointed out.

Also, I hated having 100 browser windows pop up when running integration tests, so this was something personal 🤫 🥷 🔪 😵